gen/xbuild.sh

```diff
@@ -8,51 +8,75 @@
 # compressed in an archive form (`.zip` for windows and `.tar.gz` for the rest)
 # within a folder named `dist`.
 
-set -o verbose
-
 [ -z $version ] && version=$(git describe --tags)
 
-mkdir -p dist
+if test -e lf || test -e lf.exe; then
+  echo "gen/xbuild.sh: delete lf and lf.exe from cwd before running this script"
+  exit 2
+fi
 
-# https://golang.org/doc/install/source#environment
+mkdir -p dist
```

Comment:
I wonder if it would be safer to remove `dist` and recreate it. I think it's unlikely to happen, but if it already existed and contained some files, they would end up getting released.

Reply:
I'm not sure why @gokcehan used `-p` here. I could remove it and error out if `mkdir` fails. This is more-or-less equivalent to adding `dist` to the list of files above (`lf`, `lf.exe`).

Reply:
... or maybe we should just delete it. I'm more comfortable nuking this compared to `lf`, since the `dist` directory is pretty much owned by this script.

Reply:
I guess this is pretty much the same conversation as https://github.com/gokcehan/lf/pull/1298#discussion_r1225757868 now